src/components/sections/Editor.tsx

```diff
@@ -0,0 +1,22 @@
+import { json } from '@codemirror/lang-json'
+import CodeMirror, { ViewUpdate } from '@uiw/react-codemirror'
+import React from 'react'
+
+type EditorProps = {
+    handleChange?: (value: string, viewUpdate: ViewUpdate) => void
```

Comment:
Should this be optional?  It seems quite necessary for this component to function?